applications/FluidDynamicsApplication/custom_strategies/schemes/bdf2_turbulent_scheme.h

```diff
@@ -81,17 +79,15 @@ class GearScheme : public Scheme<TSparseSpace, TDenseSpace>
     typedef Dof<TDataType> TDofType;
     typedef typename BaseType::DofsArrayType DofsArrayType;
 
-
-
     ///@}
     ///@name Life Cycle
     ///@{
 
     /// Default constructor.
-    GearScheme()
-        :
-        Scheme<TSparseSpace, TDenseSpace>(),
-        mrPeriodicIdVar(Kratos::Variable<int>::StaticObject())
+    BDF2TurbulentScheme(const unsigned int DomainSize)
```

Comment:
why passing DOMAIN_SIZE from outside and not using the ProcessInfo like we usually do?

Reply:
Makes sense. I'll try to remove it.

Reply:
Now I see. The point is that at construction time the `scheme` doesn't know about the model part. Hence we need to provide the domain size as an argument to set up the corresponding rotation tool.

Reply:
Maybe we can set up the rotation tool in the initialize to remove this. :thinking:

Reply:
Done.